### PR TITLE
How to install and configure graylog2 on debian 9

### PR DESCRIPTION
**Pull request by hitjethva**
This is my first guide generated for Linode Bounty program.

### CODE REVIEW

File: how-to-install-and-configure-graylog2-on-debian-9.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+    [![Graylog Dashboard](/docs/assets/Screenshot-of-graylog-dashboard_small.png)](/docs/assets/Screenshot-of-graylog-dashboard.png)
+
+3.  Next, you will need to configure Graylog Input to receive the logs from the external source. To do so, click on the System > Inputs. Then select Syslog UDP from this dropdown, and then click on the **Launch new input** button. You should see the following image:
+
+    [![Graylog Add Input](/docs/assets/Screenshot-of-graylog-syslog-input_small.png)](/docs/assets/Screenshot-of-graylog-syslog-input.png)
+
+4.  Here, fill all the details then click on the **Save** button. You should see the local input in the following image:
+
+    [![Graylog Input Dashboard](/docs/assets/Screenshot-of-graylog-input-dashboard_small.png)](/docs/assets/Screenshot-of-graylog-input-dashboard.png)
```
[the remark this answers is not in the document]
Comment (hitjethva) [reply how-to-install-and-configure-graylog2-on-debian-9.md]: Ok

**Comment by GuessWhoSamFoo** [comment]
@hitjethva You need to implement changes per review comments which then gets approved before we can move forward with this document.

**Comment by hitjethva** [comment]
@GuessWhoSamFoo;

i have made all  the changes as per your comments.

**Comment by GuessWhoSamFoo** [comment]
Closed for internal editing. See above for new PR.

**Pull request closed without merging**